packages/addon-dev/src/rollup-app-reexports.ts

```diff
@@ -13,7 +13,10 @@ export default function appReexports(opts: {
       let pkg = readJsonSync('package.json');
       let appJS: Record<string, string> = {};
       for (let filename of Object.keys(bundle)) {
-        if (opts.include.some((glob) => minimatch(filename, glob))) {
+        if (
```

Comment:
I kinda like the idea of being able to have .d.ts files co-located with the js and js.map files!

Reply:
Even for the app files that get's merged into the host app? You can still have colocated in the other files, not in the app folder.

Reply:
oh, I mean I support this change! I'm excited for it!
I've been making separate types directories because of the way it used to be